packages/terra-form-select/src/native-select/NativeSelect.jsx

```diff
@@ -216,14 +216,14 @@ const NativeSelect = ({
     currentValue = getFirstValue(options, isFilterStyle);
     currentDisplay = getDisplay(currentValue, options, isFilterStyle, intl);
   }
-
+  const selectedLabel = intl.formatMessage({ id: 'Terra.form.select.selected' });
   const selectAttrs = {
     'aria-describedby': ariaDescribedBy,
-    'aria-label': ariaLabel,
+    'aria-label': (uncontrolledValue || value !== undefined) ? `${ariaLabel}, ${selectedLabel}` : ariaLabel,
     id,
     disabled,
     'aria-invalid': isInvalid || undefined,
-    required,
+    'aria-required': required,
```

Comment:
Would removing the actual `required` attribute allow the select tag to submit? Shouldn't the original `required` attribute still be here alongside `aria-required`?

Reply:
I think we should revert this change and keep the `required` attribute as it servers both purpose of preventing form submit and helps screen reader to identify field as required

Reply:
reverted as suggested.Thanks